lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -33,6 +33,49 @@
 namespace flutter {
 
 namespace {
+
+#ifdef FML_OS_ANDROID
```

Comment:
This doesn't actually need to be conditionally compiled, there's nothing here that requires android.  The usage is where the branching should happen.  The less conditional compilation, the better.